hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/CalciteSqlOptimizer.java

```diff
@@ -352,13 +353,27 @@ private SqlPlan toCreateMappingPlan(PlanKey planKey, SqlCreateMapping sqlCreateM
         List<MappingField> mappingFields = sqlCreateMapping.columns()
                 .map(field -> new MappingField(field.name(), field.type(), field.externalName()))
                 .collect(toList());
-        Mapping mapping = new Mapping(
-                sqlCreateMapping.nameWithoutSchema(),
-                sqlCreateMapping.externalName(),
-                sqlCreateMapping.type(),
-                mappingFields,
-                sqlCreateMapping.options()
-        );
+
+        Mapping mapping;
+        if (nodeEngine.getVersion().asVersion().isLessThan(V5_3)) {
```

Comment:
Jet will not execute jobs (eg. SELECTs) unless the cluster is fully upgraded to the same version as the code. Do we want to support DDL during cluster upgrade (in particular rolling upgrade)?

Reply:
I think it's OK to support them. It's not that we must not run them, but we'd like to, but we can't. We can run this command, so why fail?

But this check is wrong. We must fail if version is less than 5.3, not create a broken datalink.